src/test/java/server/IntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+package server;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class IntegrationTest {
+    Map<String, Handler> routes = Map.of(
+            "GET /", new RootPathHandler(),
+            "GET /simple_get", new SimpleGetHandler(),
+            "HEAD /simple_get", new SimpleGetHandler(),
+            "GET /simple_get_with_body", new SimpleGetWithBodyHandler(),
+            "HEAD /head_request", new HeadRequestHandler(),
+            "GET /redirect", new RedirectHandler()
+    );
```

Comment:
**question** Does this duplicate an idea that we have elsewhere?

Reply:
It looks like this idea is being duplicated each time I need to create a new Router (in HTTPServerMain, "IntegrationTest", and RouterTest). 

My first thought is to extract a record to represent the idea of Routes, and then it would be flexible to hold the routes for any application. 

But I think that would still mean duplicating the same map in HTTPServerMain and in the integration test. Unless the user passed in their own map of routes as an argument when they started up the server, but that seems cumbersome for them to have to do.  

Not sure I'm on the right track here - I feel like I may be missing something simple!